.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -2,40 +2,18 @@ name: Publish docs to Wiki
 
 on:
   push:
-    paths:
-      - docs/**
     branches:
-      - main
-
-env:
-  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }}
-  USER_NAME: team-nimblehq
-  USER_EMAIL: dev@nimblehq.co
-  OWNER: ${{ github.event.repository.owner.name }}
-  REPOSITORY_NAME: ${{ github.event.repository.name }}
+      - chore/230-use-publish-github-wifi-action-in-publish-docs-to-wiki-workflow
 
 jobs:
   publish_docs_to_wiki:
     name: Publish docs to Wiki
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
-      - name: Checkout source code
-        uses: actions/checkout@v2
-
-      - name: Pull content from Wiki
-        run: |
-          mkdir tmp_wiki
-          cd tmp_wiki
-          git init
-          git config user.name $USER_NAME
-          git config user.email $USER_EMAIL
-          git pull https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git
-
-      - name: Push content to Wiki
-        run: |
-          rsync -av --delete docs/ tmp_wiki/ --exclude .git
-          cd tmp_wiki
-          git add .
-          git commit -m "Update Wiki content"
-          git push -f --set-upstream https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git master
+      - name: Publish Github Wiki
+        uses: nimblehq/publish-github-wiki-action@v0.1.0-alpha-3
```

Comment:
should it be `nimblehq/github-actions-workflows/.github/workflows/publish_wiki.yml@0.1.0`?

Reply:
oh `nimblehq/publish-github-wiki-action@v0.1.0-alpha-3` works 👍